src/jobs/config/jobConfig.createExample.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "jobType": "archive",
+    "create": {
+      "actions": [
+        {
+          "actionType": "log"
+        },
+        {
+          "actionType": "email",
+          "mailer": {
+            "host": "smtp.forwardemail.net",
+            "port": 465,
+            "secure": true,
+            "auth": {
+              "user": "REPLACE-WITH-YOUR-ALIAS@forwardemail.net",
+              "pass": "REPLACE-WITH-YOUR-GENERATED-PASSWORD"
```

Comment:
I wonder if this is the right thing to do?
Do we want to encourage admins to have a clear text password here?

Reply:
This is up for discussion. It is currently just an example of what the config of an email action could look like, but we can very easily change it.

Reply:
What's the correct way to do this? Is it any different than having cleartext in `.env`?

Reply:
it depends from the target system. 
I would strongly discourage to have this syntax in an example in core, as mindless users might mistakenly leave their password in files

Reply:
Let's table the issue for now and review it later